libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -416,9 +417,11 @@ bool AP_Arming::ins_gyros_consistent(const AP_InertialSensor &ins)
 
     return true;
 }
+#endif // AP_INERTIALSENSOR_ENABLED
 
 bool AP_Arming::ins_checks(bool report)
 {
+#if AP_INERTIALSENSOR_ENABLED
```

Comment:
This should be around the entire method, not in the body here.  You can *not* do the `#endif` just above this method and instead do it below this method...

This is a pattern we're already using for various other libraries in here.

Reply:
reworked, please check again